elasticdl/python/worker/worker.py

```diff
@@ -112,6 +122,7 @@ def __init__(
         model_inst = model_handler.get_model_to_train(model_inst)
 
         self._embedding_service_endpoint = embedding_service_endpoint
+        # self.set_model will use self._use_multi_ps
```

Comment:
Remove this comment as the code is self-explainatory

Reply:
Done